projects/core/src/checkout/store/effects/checkout.effect.ts

```diff
@@ -256,14 +256,18 @@ export class CheckoutEffects {
     | CheckoutActions.PlaceOrderSuccess
     | GlobalMessageActions.AddMessage
     | CheckoutActions.PlaceOrderFail
+    | CartActions.RemoveCart
   > = this.actions$.pipe(
     ofType(CheckoutActions.PLACE_ORDER),
     map((action: any) => action.payload),
     mergeMap(payload => {
       return this.checkoutConnector
         .placeOrder(payload.userId, payload.cartId)
         .pipe(
-          switchMap(data => [new CheckoutActions.PlaceOrderSuccess(data)]),
+          switchMap(data => [
+            new CartActions.RemoveCart(payload.cartId),
```

Comment:
Why we added this? Is it related to the refactor or it's a bugfix for sth else?

Reply:
Previously logic for clearing whole cart state was in meta reducer. In multi cart state it doesn't make sense to clear the whole state, so it was moved to this effect instead.